tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -2203,6 +2186,179 @@ public void handlePath(String xpath) {
                 seen.addAll(paths);
             }
         });
+
+        // 'R' = Revert to baseline version under certain conditions
+        fixList.add('R', "Revert under certain conditions", new CLDRFilter() {
+            // vxmlDir needs to be the "plain" (without post-processing) path of an existing copy of common/main
+            // For example, vetdata-2023-01-23-plain-dropfalse ... see https://github.com/unicode-org/cldr/pull/2659
+            // Also ldml.dtd is required -- and should already have been created by ST when generating vxml
+            final private String vxmlDir = "../vetdata-2023-01-23-plain-dropfalse/vxml/";
+            final private File[] list = new File[]{
+                new File(vxmlDir + "common/main/"),
+                new File(vxmlDir + "common/annotations/")
```

Comment:
```suggestion
                new File(vxmlDir + sourceInput)
```

fyi @macchiati @btangmu  — otherwise, vxmlFile has both, main and annotations, instead of just the dir actually under consideration.

Reply:
good

Reply:
It's still not right.   `[SOURCEDIR].value` is not parsed yet at this stage of the code.

Reply:
see sub-PR.